web/index.html

```diff
@@ -4,7 +4,38 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta
+      name="description"
+      content="This website is version 2 of My-Cv which it's uses just PURE HTML5 and CSS3"
+    />
+    <link rel="stylesheet" href="css/style.css" />
     <title>My CV V2</title>
   </head>
-  <body></body>
+  <body>
+    <div class="container">
+      <h1>Bogdan Negura</h1>
+      <h2>Junior Web Developer</h2>
+      <h3>About Me</h3>
+      <section class="section">
```

Comment:
I believe the section should contain the previous h1, h2, h3 tags as well. It's the intro / welcome section with everything in it.